contracts/xchain/PolygonChild.sol

```diff
@@ -3,11 +3,12 @@ pragma solidity ^0.8.0;
 
 import "@fx-portal/contracts/tunnel/FxBaseChildTunnel.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
+import "../contracts/coordination/ITACoChildToRoot.sol";
 
-contract PolygonChild is FxBaseChildTunnel, Ownable {
+contract PolygonChild is ITACoChildToRoot, FxBaseChildTunnel, Ownable {
     event MessageReceived(address indexed sender, bytes data);
```

Comment:
This event was removed in a previous commit of my `sync` branch, since the bridge can't process events in the `_processMessageFromRoot` callback (see commit b2ea8d9d4470802de0c82efd4660c7c589a09e74 description)